content/nginx-one/changelog.md

```diff
@@ -30,6 +30,22 @@ h2 {
 
 Stay up-to-date with what's new and improved in the F5 NGINX One Console.
 
+## Apr 28, 2025
+
+### Manage RBAC access with namespaces
+
+We have added support for namespaces in N1C. You can now:
+
+- Manage resources in isolation in different namespaces.
+- Configure granular user permission controls base on namespace.
+
+If you already have set up resources in a namespace that:
+
+- Does not exist in your F5 Distributed Cloud tenant
+- Was deleted from your NGINX One Console tenant
+
+Then you may lose access to those resources. To regain access, create the matching namespace in your F5 Distributed Cloud (F5XC) tenant, and assign necessary [RBAC](https://docs.cloud.f5.com/docs-v2/administration/how-tos/user-mgmt/roles) permissions to access that namespace.
+
```

Comment:
Since this problem won't happen until the _next_ XC release

```suggestion
```

Opened https://github.com/nginx/documentation/issues/462 to support.